.travis.yml

```diff
@@ -26,9 +27,7 @@ before_deploy:
 deploy:
   provider: script
   script: yarn lerna publish --yes
-  email: marianneb@nulogy.com
```

Comment:
Ended up cleaning these up while trying to fix the memory leak issue but they had nothing to do with it.  I just noticed that Travis had a notification that these fields were unknown or deprecated anyways.